frontend/src/components/AdminSuspendModal/AdminSuspendModal.js

```diff
@@ -31,18 +31,14 @@ function AdminSuspendModal(props) {
     <div>
       <Modal className="suspend-user-modal" isOpen={suspendModal} toggleSuspendModal={toggleSuspendModal} backdrop={false}>
         <ModalHeader toggleSuspendModal={toggleSuspendModal} close={closeBtn}>
-          Suspend User?
+          { selectedUser.is_suspended ? 'Unsuspend User?' : 'Suspend User?' }
         </ModalHeader>
         <ModalBody>
-          Clicking &quot;Suspended&quot; will prevent the user from logging into the app.
-
-            The user will remain suspended until this status is removed by an administrator.
-
-            If you do not wish to suspend the user, close this dialog box.
+          { selectedUser.is_suspended ? 'Clicking "UnSuspend" will allow the user to logging into the app. The user will remain unsuspended until this status is removed by an administrator.If you do not wish to unsuspend the user, close this dialog box.' : 'Clicking "Suspended" will prevent the user from logging into the app. The user will remain suspended until this status is removed by an administrator.If you do not wish to suspend the user, close this dialog box.' }
```

Comment:
This should definitely be broken out into multiple lines. Probably worth turning it into a function or something, since the text is guaranteed to go over the line width limit.